docs/src/pages/components/menu/demo.tsx

```diff
@@ -1,37 +1,61 @@
 import * as React from 'react';
+import { Divider, Menu, MenuItem } from '@aws-amplify/ui-react';
 
-import { Divider, Menu, MenuItem, View } from '@aws-amplify/ui-react';
-import { MenuPropControls } from '@/components/MenuPropControls';
-import { useMenuProps } from '@/components/useMenuProps';
-import { Example } from '@/components/Example';
+import { Demo } from '@/components/Demo';
+import { MenuPropControls, MenuPropControlsProps } from './MenuPropControls';
+import { useMenuProps } from './useMenuProps';
+
+const propsToCode = (props: MenuPropControlsProps) => {
+  const size = props.size ? `\n  size=${props.size}` : '';
+  return `<Menu ${size}
+  menuAlign={${props.menuAlign}}
```

Comment:
Can we demonstrate passing this prop as a string instead of as a variable?

`menuAlign="${props.menuAlign}"`

Reply:
good catch, will update